website_menu_icon/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module allows to assign image or icon to individual menu entries.
+These can be used in a custom theme to render menus with icons.
+The module also brings back option for opening menu item in new window
```

Comment:
Well it is and is not unrelated at the same time. Without this the only way to have menu opened in new window is to change the attribute in the backend which seems somewhat not-intuitive. So I wanted to be able to configure all apsects of the menu here in the same place. It could be taken into it's own separate module but that seems to me somewhat not worth doing (separate module just for this).